src/Domain.Schema/Extentions/StringExtensions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace OMP.Connector.Domain.Schema.Extentions
```

Comment:
Spelling mistake in "Extentions"
Should be Extensions

Should fix file name too.

Reply:
Done